index.html

```diff
@@ -3724,25 +3723,26 @@ <h3><dfn>WebPublicationManifest</dfn> dictionary </h3>
 			</section>
 
 			<section id="contributor-idl">
-				<h3><dfn>Person</dfn> and <dfn>Organization</dfn> members</h3>
+				<h3><dfn>Contributor</dfn>  members</h3>
 
-				<p>These definitions reflect the basic information derived from the schema.org <a
-						href="https://schema.org/Person">Person</a> and <a
-						href="https://schema.org/Organization">Organization</a> classes,
-					respectively. The WebIDL definitions only contain the minimal information for
-					the <a>infoset</a>; user agents MAY interpret a wider range of properties, as
-					defined by schema.org.</p>
+				<p>
+					These definitions reflect the basic information derived from the schema.org <a href="https://schema.org/Person">Person</a> and <a href="https://schema.org/Organization">Organization</a> classes. The WebIDL definitions only contain the minimal information for the <a>infoset</a>; user agents MAY interpret a wider range of properties, as defined by schema.org.
+				</p>
 
 
 				<p>The <a href="#dom-webpublicationmanifest-artist">artist</a>, <a
 						href="#dom-webpublicationmanifest-author">author</a>, etc., members are each
-					a sequence of <a><code>Person</code></a> or <a><code>Organization</code></a>
-					dictionaries with the following members:</p>
+					a sequence of <a><code>Contributor</code></a> dictionaries, whose <code>type</code> member
+					indicate whether the contributor is a <a href="https://schema.org/Person">Person</a> or an <a href="https://schema.org/Organization">Organization</a>. The members of this dictionary are:</p>
```

Comment:
This is grammatically awkward, as it switches from a plural sequence to suggesting there is only one type set. I would think this should be "each of whose type member indicates whether the contributor is a ..."

Reply:
Done (commit follows). Thanks!